gov-input/anvil/new-ERC20Transfer.json

```diff
@@ -2,8 +2,8 @@
   "chainid": "31337",
   "description": "Transfer treasury token",
   "network": "anvil",
-  "protocolToken": "0x0000000000000000000000000000000000000000",
-  "odGovernor": "0xA5DC5C86212cF22167c52b756363B0FeA7B6e591",
+  "ProtocolToken_Address": "0x0000000000000000000000000000000000000000",
```

Comment:
If we are pulling from the Contracts.s.sol files, why do we need to include the addresses in the input jsons?

Reply:
I just had this here to test that the addresses were successfully being swapped out.  the way I have it set up is any address  in the json with the same name as an address in the contracts.s.sol will get swapped out.  if it has a different name it will be left alone.

Reply:
Hmmm. I think that could be confusing if I enter the address in the .json, but then the output file it gets modified to another address.

Reply:
so I changed these to be empty strings.  the script will add the addresses in the output but leave the input addresses empty when -g is ran.